src/rocksdb.rs

```diff
@@ -513,6 +513,71 @@ impl DB {
         self.get_cf_opt(cf, key, &ReadOptions::new())
     }
 
+    pub fn multi_get_cf_opt(&self,
+                            cf: &CFHandle,
+                            keys: &[&[u8]],
```

Comment:
Looks like using generic type here is more suitable. like `f<F: AsRef<[u8]>>(keys: &[F])`